profiles/apparmor.d/usr.share.openqa.script.worker

```diff
@@ -100,7 +100,7 @@
   /usr/sbin/ipmiconsole rix,
   /usr/share/openqa/lib/DBIx/Class/Timestamps.pm r,
   /usr/share/openqa/lib/OpenQA/** r,
-  /usr/share/openqa/lib/db_helpers.pm r,
+  /usr/share/openqa/lib/** r,
```

Comment:
I think this covers also the 2 lines above, but let's leave it as is

Reply:
Yup.. just in case 👽